extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -131,6 +133,13 @@ function VideoWrapperWithRef(
     };
   }, [mediasession, playing, metadata, play, pause]);
 
+  // Pause if the video goes into a "paused" context.
+  useEffect(() => {
+    if (!playable) {
```

Comment:
Does this need to check `!playable && playing`?

Reply:
Elsewhere it looked like calling `pause()` without checking is ok, so I hoped this could be simpler here as well.

Reply:
We should see how other player components implement this. Are we leaving it up to the implementer to guard if necessary? For example I wonder if a video has not started playing yet, what the effect of `pause` will be. Same with `autoplay` - should we pause a autoplay in a "paused" context? Will that change the interactivity state on the video to imply a user has done something?

Reply:
Yeah. These are good questions. But I'd like @alanorozco to help out here.